src/pages/app/helper.ts

```diff
@@ -18,7 +22,9 @@ const useAppPrepareHook = (): IAppPrepareHook => {
       const data = await getUsers();
       setUsers(data);
     } catch (error) {
+      const err = error as FirebaseError;
```

Comment:
Do not like this because you cannot confirm it's an firebase error. You can use instanceof to check and handle for case handling firebase error.